RULES.md

```diff
@@ -0,0 +1,93 @@
+This document pertains to the entire LibreELEC tree and all of it's branches.
+
+#### Table of Contents
+
+- [Package Version Bumps](https://github.com/LibreELEC/LibreELEC.tv/blob/master/RULES.md#package-version-bumps)
+  - [LibreELEC Testing Window](https://github.com/LibreELEC/LibreELEC.tv/blob/master/RULES.md#libreelec-testing-window)
+  - [LibreELEC Release Window](https://github.com/LibreELEC/LibreELEC.tv/blob/master/RULES.md#libreelec-release-window)
+  - [Major Packages](https://github.com/LibreELEC/LibreELEC.tv/blob/master/RULES.md#major-packages)
+  - [Minor Packages](https://github.com/LibreELEC/LibreELEC.tv/blob/master/RULES.md#minor-packages)
+- [Fixes](https://github.com/LibreELEC/LibreELEC.tv/blob/master/RULES.md#fixes)
+- [Reverts](https://github.com/LibreELEC/LibreELEC.tv/blob/master/RULES.md#reverts)
+- [Trust](https://github.com/LibreELEC/LibreELEC.tv/blob/master/RULES.md#trust)
+- [Changes to RULES.md](https://github.com/LibreELEC/LibreELEC.tv/blob/master/RULES.md#changes-to-mastermd)
+
+## Package Version Bumps
+
+If a package version is broken it must be signified with a comment stating what is broken and possibly a linked bug report.
+```
+# version 1.23 is broken on some systems, see https://bug.report.com/123456
+```
+
+#### LibreELEC Testing Window
+
+Anything listed in the [Major Packages](https://github.com/LibreELEC/LibreELEC.tv/blob/master/RULES.md#major-packages) section **MUST** be submitted as a PR and reviewed.
```

Comment:
Nit: You don't need `https://github.com/LibreELEC/LibreELEC.tv/blob/master/RULES.md` on the target, so `[Major Packages](#major-packages)` should work.